docs/resources/azure_express_route_circuits.md

```diff
@@ -0,0 +1,124 @@
+| sku_tiers             | A List of  The service tier. Possible values are Basic, Local, Standard or Premium.      | `sku_tier` |
+| sku_family             | A List of  The billing mode for bandwidth. Possible values are MeteredData or UnlimitedData. | `sku_family` |
+<superscript>*</superscript> For information on how to use filter criteria on plural resources refer to [FilterTable usage](https://github.com/inspec/inspec/blob/master/dev-docs/filtertable-usage.md).
+
+
+## Examples
+
+### Ensure that the express_route_circuits resource has is from same type
+```ruby
+describe azure_express_route_circuits(resource_group: 'MyResourceGroup', name: 'bastion_name') do
+  its('type') { should eq 'Microsoft.Network/express_route_circuits' }
+end
+```
+### Ensure that the express_route_circuits resource is in successful state
+```ruby
+describe azure_express_route_circuits(resource_group: 'MyResourceGroup') do
+  its('provisioning_states') { should include('Succeeded') }
+end
+```
+
+### Ensure that the express_route_circuits resource is from same location
+```ruby
+describe azure_express_route_circuits(resource_group: 'MyResourceGroup') do
+  its('location') { should include df_location }
+end
+```
+### Test If Any express_route_circuits Exist in the Resource Group
+```ruby
+describe azure_express_route_circuits(resource_group: 'MyResourceGroup') do
+  it { should exist }
+end
+
+
+```
+
+## Matchers
+
+This InSpec audit resource has the following special matchers. For a full list of available matchers, please visit our [Universal Matchers page](https://www.inspec.io/docs/reference/matchers/).
+
+### exists
+```ruby
+# Should not exist if no express_route_circuits are in the resource group
+describe azure_express_route_circuits(resource_group: 'MyResourceGroup') do
+  its('names') { should include circuitName }
+  its('locations') { should include location }
+  its('types') { should include 'Microsoft.Network/expressRouteCircuits' }
+  its('provisioning_states') { should include('Succeeded') }
+  its('service_provider_properties_bandwidth_in_mbps') { should include bandwidthInMbps }
+  its('service_provider_properties_peering_locations') { should include peeringLocation }
+  its('service_provider_properties_names') { should include serviceProviderName }
+  its('service_provider_provisioning_states') { should include serviceProviderProvisioningState }
+  its('service_keys') { should include serviceKey }
+  its('stags') { should include stag }
+  its('global_reach_enabled') { should include globalReachEnabled }
+  its('allow_global_reach') { should include allowGlobalReach }
+  its('gateway_manager_etags') { should include gatewayManagerEtag }
+  its('allow_classic_operations') { should include allowClassicOperations }
+  its('circuit_provisioning_states') { should include circuitProvisioningState }
+  its('sku_names') { should include sku_name }
+  its('sku_tiers') { should include sku_tier }
+  its('sku_families') { should include sku_family }
```

Comment:
here we should only show the exists example since we are talking only about that